src/_gettsim/taxes/zu_verst_eink/eink.py

```diff
@@ -137,6 +138,7 @@ def eink_rente_zu_verst(
     return eink_rente_zu_verst_m * 12
 
 
+@dates_active(start="2009-01-01", change_name="sum_eink")
 def sum_eink_ohne_kapital(
```

Comment:
```suggestion
def sum_eink_ohne_kapital_eink(
```

While we are at it...